server.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+const PORT =4000;
+
+app.listen(PORT, () => {
+  console.log(`App is listening on port: ${PORT}`)
+});
```

Comment:
I'm not sure I understand the intentions for this server. Is it meant to act as the intermediary between the front end and the API servers?

Reply:
You will need a Express server running on node in order to build out (CRUD) routes for our database alongside our microservices we deploy. Did you have in mind doing it all client side?